README.md

```diff
@@ -46,34 +46,34 @@ The EximeeBPMS Rest-API is accessible through: http://localhost:8080/engine-rest
 See the [REST API](https://docs.eximeebpms.org/manual/latest/reference/rest/)
 documentation for more details on how to use it.
 
-**Note**: The REST API does not require authentication by default. Follow the instructions from the [documentation](https://docs.camunda.org/manual/latest/reference/rest/overview/authentication/)
+**Note**: The REST API does not require authentication by default. Follow the instructions from the [documentation](https://docs.eximeebpms.org/manual/latest/reference/rest/overview/authentication/)
 to enable authentication for the REST API.
 
 ## Supported tags/releases
 
 The following tag schema is used. The user has the choice between different
-application server distributions of Camunda Platform.
+application server distributions of EximeeBPMS.
 
-- `latest`, `${DISTRO}-latest`: Always the latest minor release of Camunda Platform.
+- `latest`, `${DISTRO}-latest`: Always the latest minor release of EximeeBPMS.
```

Comment:
We only support two types of tags: X.X.X and latest. {DISTRO}-latest is not supported.